proposals/0000-template.md

```diff
@@ -0,0 +1,26 @@
+* Title: *Descriptive title*
+* Champion: *@github handle/email@*
+* Conversation (optional): *thread on radicle.community*
+* References:
+  * list of
+  * external resources
+  * and prior work
+
+# Summary/Motivation/Introduction
+*High-level motivation and introduction to the problem and domain.*
+
+# Explanation
+*Longer outline in free-form and appropriate sub-headers.*
+
+# Recommendation
+* steps to move forward
+* implementation timeline/guideline (optional)
+
+# Drawbacks
+*Why should this not be done?*
+
+# Open Questions
+*Enumerate unknowns and unresolved questions.*
+
+# Future possibilities
+*Potential follow-ups and related work at a later point in time.*
```

Comment:
I would actually add one more header `# Status` which can be **Rejected**, **Accepted**, or **Pending**

Reply:
Alternatively: open PR == Pending, merged == accpeted, PR closed == rejected.

Reply:
Well I think they should all be merged to preserve this statement: https://github.com/radicle-dev/radicle-decisions/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11-R12

Reply:
You sire, played well. This battle is yours but the war is not over just yet.

***
Will amend.

Reply:
Heh :smile: :bowing_man: